src/components/progress/progress.stories.ts

```diff
@@ -31,3 +31,14 @@ export const darkThemeRTL_TestOnly = (): string => html`
 `;
 
 darkThemeRTL_TestOnly.parameters = { themes: themesDarkDefault };
+
+export const darkThemeText = (): string => html`
```

Comment:
Can you update the `darkThemeRTL_TestOnly` story above to cover your changes instead of adding a dedicated story? Since it's related to the dark theme, I think it'd make sense to set there. This would keep our snapshot quota down while covering the use case.